packages/hub/src/session.ts

```diff
@@ -119,8 +119,8 @@ export class Session implements SessionInterface {
     return dropUndefinedKeys({
       sid: `${this.sid}`,
       init: this.init,
-      started: new Date(this.started).toISOString(),
-      timestamp: new Date(this.timestamp).toISOString(),
+      started: this.started,
+      timestamp: this.timestamp,
```

Comment:
Should be sent as string, please revert.
https://develop.sentry.dev/sdk/sessions/#session-update-payload